hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -115,9 +115,10 @@ const (
 
 type NodePoolReconciler struct {
 	client.Client
-	recorder        record.EventRecorder
-	ReleaseProvider releaseinfo.Provider
-	controller      controller.Controller
+	recorder            record.EventRecorder
+	UserReleaseProvider releaseinfo.Provider
```

Comment:
Where do we set this value?

Reply:
Thanks, the previous code modification was incorrect, so I updated the code.